### DISCUSSION TITLE
Contributions doesn't added if the you not commit in main branch in your own repo

### DISCUSSION THREAD

**Discussion by hvlxh** [community/community#57743] [Repositories]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues (The feature that allows you to track ideas, feedback, tasks, or bugs for work on GitHub. Not for any issue you are having).
- [ ] Projects (For planning and tracking work on GitHub - not any open source or other type of project).

### Body

Contributions doesn't added (in the profile) if the you not commit in main branch in your own repo. Unless you merged commits into your main branch

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by danielalyoshin** [comment]
This is an intended behaviour. As stated in the [official GitHub documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#commits):

> Commits will appear on your contributions graph if they meet **all** of the following conditions:
> 
> - The email address used for the commits is associated with your account on GitHub.com.
> - The commits were made in a standalone repository, not a fork.
> - The commits were made:
>   - In the repository's default branch
>   - In the `gh-pages` branch (for repositories with project sites)
> 
> For more information on project sites, see "[About GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites)."
> 
> In addition, **at least one** of the following must be true:
> 
> - You are a collaborator on the repository or are a member of the organization that owns the repository.
> - You have forked the repository.
> - You have opened a pull request or issue in the repository.
> - You have starred the repository.

Hope this clears things up!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @hvlxh ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
